lib/bronto/contact.rb

```diff
@@ -20,11 +20,11 @@ def self.find(filter = Bronto::Filter.new, page_number = 1, fields = nil, includ
       Array.wrap(resp[:return]).map { |hash| new(hash) }
     end
 
-    def self.save(*objs)
+    def self.save(endpoint, *objs)
```

Comment:
@jaemo after thinking about this more, it'd be ideal to not change the signature of this public method. My thinking is that we don't really want to maintain this library and this is a clear, breaking change. Could you try to refactor the internal implementation of this to extract out a private method that `save` and `update` can share?